src/packages/files/files.ts

```diff
@@ -89,6 +90,59 @@ export default (app: Express) => {
     }
   });
 
+  /**
+   * @swagger
+   * /file/{id}:
+   *   get:
+   *     parameters:
+   *       - description: ID of the file
+   *         in: path
+   *         name: id
+   *         required: true
+   *         schema:
+   *           type: number
+   *     produces:
+   *       - application/json
+   *     responses:
+   *       200:
+   *         description: Fetches the file by id
+   *     summary: Get the file with the associated ID.
+   *     tags:
+   *       - Files
+   */
+  app.get('/file/:id', filesLimiter, auth, async (req, res) => {
+    const { params } = req;
+    const { id } = params;
+
+    try {
+      const { rows, rowCount } = await db.query(
+        `
+        select *
+        from merlin.uploaded_file
+        where id = $1
+        and deleted_date is null;
+      `,
+        [id],
+      );
+
+      if (rowCount && rowCount > 0) {
+        const [row] = rows;
+        const { name } = row;
+
+        const filePath = path.join(FILE_PATH, name);
+
+        res.sendFile(filePath, err => {
+          if (err) {
+            logger.info(`GET /file/{id}: No file was found with id: ${id}`);
+          }
+        });
+      }
+    } catch (error: any) {
+      logger.error(error);
+      res.status(404).json({ message: error.message, success: false });
```

Comment:
Please flatten this response into just the `error.message`, not wrapped in a JSON. The `success` flag is not present in the 200 response, which makes the error and success structures inconsistent. Additionally, the user already knows that the request failed because of the 404 code, so the variable isn't communicating any new information.